src/ChunkData.h

```diff
@@ -96,23 +99,14 @@ class cChunkData
 	Allows a_Src to be nullptr, in which case it doesn't do anything. */
 	void SetSkyLight(const NIBBLETYPE * a_Src);
 
-	struct sChunkSection
-	{
-		BLOCKTYPE  m_BlockTypes   [SectionHeight * 16 * 16]    ;
-		NIBBLETYPE m_BlockMetas   [SectionHeight * 16 * 16 / 2];
-		NIBBLETYPE m_BlockLight   [SectionHeight * 16 * 16 / 2];
-		NIBBLETYPE m_BlockSkyLight[SectionHeight * 16 * 16 / 2];
-	};
+	/** Returns the number of sections present (i.e. non-air). */
+	UInt32 Sections() const;
```

Comment:
`NumSections` is already used for the total number of sections per chunk.  How about `NumPresentSections`?